Remove contributors file from NVDA, read license as HTML (#17600)
nvaccess/nvda: user_docs/en/changes.md

  * Updated CLDR to version 46.0. (#17484, @OzancanKaratas)
* Short versions of the most commonly used command line options have been added: `-d` for `--disable-addons` and `-n` for `--lang`.
Prefix matching on command line flags, e.g. using `--di` for `--disable-addons` is no longer supported. (#11644, @CyrilleB79)
* The "Contributors" file was removed from the NVDA menu. (#16922)

> * The NVDA license is now formatted as an HTML browseable message to improve readability. (#17600)

Author: ```suggestion
* The NVDA license is now formatted in HTML to improve readability. (#17600)
```
